src/dialogs/more-info/controls/more-info-configurator.ts

```diff
@@ -0,0 +1,128 @@
+import "@material/mwc-button";
+import { HassEntity } from "home-assistant-js-websocket";
+import { css, html, LitElement } from "lit";
+import { customElement, property, state } from "lit/decorators";
+import "../../../components/ha-alert";
+import "../../../components/ha-circular-progress";
+import "../../../components/ha-markdown";
+import "../../../components/ha-textfield";
+import type { HomeAssistant } from "../../../types";
+
+@customElement("more-info-configurator")
+export class MoreInfoConfigurator extends LitElement {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public stateObj?: HassEntity;
+
+  @state() private _isConfiguring = false;
+
+  private _fieldInput = {};
+
+  render() {
+    if (this.stateObj?.state !== "configure") {
+      return html``;
+    }
+
+    return html`
+      <div class="container">
+        <ha-markdown
+          breaks
+          .content=${this.stateObj.attributes.description}
+        ></ha-markdown>
+
+        ${this.stateObj.attributes.errors
+          ? html`<ha-alert alert-type="error">
+              ${this.stateObj.attributes.errors}
+            </ha-alert>`
+          : ""}
+        ${this.stateObj.attributes.fields.map(
+          (field) => html`<ha-textfield
+            .label=${field.name}
+            .name=${field.id}
+            .type=${field.type}
+            @change=${this._fieldChanged}
+          ></ha-textfield>`
+        )}
+        ${this.stateObj.attributes.submit_caption
+          ? html`<p class="submit">
```

Comment:
Does this need to be a paragraph element

Reply:
No, could be a div, but 🤷‍♂️  (it is used to center the button)